spec/factories/finance/statements.rb

```diff
@@ -21,6 +21,10 @@
 
     factory :ecf_statement, class: "Finance::Statement::ECF" do
       cpd_lead_provider { association :cpd_lead_provider, :with_lead_provider }
+
+      factory :ecf_paid_statement, class: "Finance::Statement::ECF::Paid" do
```

Comment:
I'd argue that `create(:ecf_statement, :paid)` should be sufficient and adding an extract factory isn't necessarily worth it.

Reply:
I added this factory to be inline with the `npq_paid_statement`, however I think we need it due to the class specified, where the `paid` trait does not supply that